status-network-contracts/test/RewardsStreamerMP.t.sol

```diff
@@ -3646,3 +3646,67 @@ contract FuzzTests is StakeManagerTest {
         assertEq(karma.balanceOfRewardDistributor(address(streamer)), streamerKarmaBalanceBefore - redeemed);
     }
 }
+
+contract VotesTest is StakeManagerTest {
+    uint256 public rewardAmount;
+
+    function setUp() public virtual override {
+        super.setUp();
+
+        rewardAmount = 10e18;
+        uint256 stakeAmount = 50e18;
+        uint256 rewardPeriod = 1 weeks;
+
+        assertEq(karma.balanceOf(alice), 0);
+        assertEq(karma.getVotes(alice), 0);
+
+        _stake(alice, stakeAmount, 0);
+        _setRewards(rewardAmount, rewardPeriod);
+        vm.warp(vm.getBlockTimestamp() + rewardPeriod);
+
+        assertEq(karma.balanceOf(alice), rewardAmount);
+        assertEq(karma.getVotes(alice), 0);
+    }
+
+    function checkVotesAndRedeem(address account, uint256 amount) public { }
+
+    function test_delegateBeforeRedeemingCreatesCheckpointsWithoutVotes() public {
+        vm.prank(alice);
+        karma.delegate(alice);
+
+        assertEq(karma.balanceOf(alice), rewardAmount);
```

Comment:
Maybe adding a comment here that, while we have `delegate()`ed, the karma from `setup()` isn't converted yet, therefore `getVotes()` is expected to be `0`